devops/pipeline/e2etest.yaml

```diff
@@ -162,14 +162,12 @@ stages:
           itemPattern: $(packagePattern)
 
       - script: |
-          cp `find . -name '*.deb'` $(Build.SourcesDirectory)/devops/terraform/host/osconfig.deb
+          cp `find . -name '*.deb' | head -n 1` $(Build.SourcesDirectory)/devops/terraform/host/osconfig.deb
```

Comment:
Is this a direct result of adding the signed packages, or was this also possible before?

Reply:
I'm curious what the artifact name for signed vs unsigned looks like?

Reply:
its a direct result of adding them. We stage the package based on the `packagePattern` but this pattern now matches TWO packages, the signed and the unsigned package ex using pattern "**\*bionic_x86_64"
```
./OSConfig_1.0.1.2022030701_SIGNED/osconfig_1.0.1.20220307_bionic_x86_64.deb
./OSConfig_1.0.1.2022030701_ubuntu18.04_amd64/osconfig_1.0.1.20220307_bionic_x86_64.deb
```